<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Weather</class>
 <widget class="QDialog" name="Weather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>278</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Метеоумови</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="cloudness_label">
          <property name="text">
           <string>Хмарність</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cloudness_box">
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="temperature_label">
          <property name="text">
           <string>Температура</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="temperature_spin_box">
          <property name="minimum">
           <number>-20</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
          <property name="value">
           <number>-10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="wind_speed_label">
          <property name="text">
           <string>Швидкість вітру</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="wind_speed_spin_box">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="wind_direction_label">
          <property name="text">
           <string>Напрям вітру</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="wind_direction_spin_box">
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="value">
           <number>180</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="random_button">
        <property name="text">
         <string>Згенерувати випадкові дані</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QDialogButtonBox" name="dialog_button_box">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
        <property name="centerButtons">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
